applications/nrf5340_audio/src/audio/le_audio_rx.c

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <nrfx_clock.h>
```

Comment:
It seems that zephyr/kernel.h should be included here, it is probable included through some of the other files, but we can't rely on that being the case in the future.